package.json

```diff
@@ -3,12 +3,18 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./public/serve"
+    "start": "node ./public/serve",
+    "dev":"nodemon ./public/serve"
   },
   "dependencies": {
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
+    "dotenv": "^10.0.0",
     "express": "~4.16.1",
-    "morgan": "~1.9.1"
+    "morgan": "~1.9.1",
+    "mysql": "^2.18.1"
```

Comment:
It's possible for us to work like this, but this is not a great idea. For the sake of consistency, easy and convenience it's better to work with `ORM`s or `ODM`s for large projects and even small ones. Do carryout more research on Database `ORM`s and `ODM`s.

Also see : [Top 11 Node.js ORMs, Query Builders & Database Libraries in 2021](https://www.prisma.io/dataguide/database-tools/top-nodejs-orms-query-builders-and-database-libraries)

My top recommendations would be
- [TypeORM](https://github.com/typeorm/typeorm)
- [Sequelize](https://github.com/sequelize/sequelize/)
- [Prisma](https://github.com/prisma/prisma)

Please integrate and `ORM` instead, preferably `TypeORM`. Thanks!

Reply:
Okay sir I'll work on that this night